OpenRA.Mods.Common/Widgets/Logic/ReplayBrowserLogic.cs

```diff
@@ -93,7 +94,7 @@ public ReplayBrowserLogic(Widget widget, ModData modData, Action onExit, Action
 			Ui.LoadWidget("MAP_PREVIEW", mapPreviewRoot, new WidgetArgs
 			{
 				{ "orderManager", null },
-				{ "getMap", (Func<MapPreview>)(() => map) },
+				{ "getMap", (Func<(MapPreview, ServerMapStatus)>)(() => (map, ServerMapStatus.Playable)) },
```

Comment:
Same here.

Reply:
Done at the receiving side. I don't think there is any value in adding them here since the context is immediately obvious from the lambda, but can change this if you insist.

Reply:
Fair enough, we can keep it as is here.